main.py

```diff
@@ -1,21 +1,21 @@
-# Original Python game script
+# Original Segun Adebayo's game script
 import random
 
-def play_game():
+def play_game(): Segun Adebayo's first Effort
     print("Welcome to the Number Guessing Game!")
     secret_number = random.randint(1, 100)
```

Comment:
I want to you to edit edit the game rule and secret_number should be a number between 1 and 50 not 100.